release-notes.md

```diff
@@ -46,7 +48,7 @@ What's new in TimescaleDB 2.0:
 - **New and improved informational views**: Get more insight about hypertables,
   chunks, policies, and job scheduling.
 
->:TIP:As TimescaleDB 2.0 is currently available as a release candidate, we encourage
+>:TIP: TimescaleDB 2.0 is currently GA, we encourage
```

Comment:
Cut this Tip?  It's mainly about it being an RC, not GA.

Reply:
You are completely right.